Issues with assembleAndroidTest Task on 5.0.0

## Issue

I use Detox as the e2e framework for my react native app, and on android the test class is compiled using gradle's `assembleAndroidTest` task. After updating to 5.0.0 that task fails with the following error.
```
> Transform artifact browser.aar (androidx.browser:browser:1.2.0) with DexingTransform
AGPBI: {"kind":"error","text":"Static interface methods are only supported starting with Android N (--min-api 24): androidx.browser.trusted.TrustedWebActivityDisplayMode androidx.browser.trusted.TrustedWebActivityDisplayMode.fromBundle(android.os.Bundle)","sources":[{}],"tool":"D8"}

> Task :react-native-app-auth:mergeExtDexDebugAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-app-auth:mergeExtDexDebugAndroidTest'.
> Could not resolve all files for configuration ':react-native-app-auth:debugAndroidTestRuntimeClasspath'.
   > Failed to transform artifact 'browser.aar (androidx.browser:browser:1.2.0)' to match attributes {artifactType=android-dex, dexing-is-debuggable=true, dexing-min-sdk=16}.
      > Execution failed for DexingTransform: /Users/akoon/.gradle/caches/transforms-2/files-2.1/4d2f0248e5674aca5a403ca4d18d323c/jars/classes.jar.
         > Error while dexing.
```
The fix is described [here](https://stackoverflow.com/questions/49512629/default-interface-methods-are-only-supported-starting-with-android-n). After adding the suggestion to the `build.gradle` file I was able to run the `assembleAndroidTest` task successfully.

---

## Environment

* **Your Identity Provider**: `Keycloak`
* **Platform that you're experiencing the issue on**: `Android`
* **Are you using Expo?** No

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -34,6 +34,10 @@ android {
     lintOptions {
         abortOnError false
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 repositories {

